etl/steps/data/garden/demography/2022-12-08/population/meta.yml

```diff
@@ -59,13 +63,18 @@ tables:
         description: |
           Share of the world's population by country, available from 10,000 BCE to 2100 based on Gapminder data, HYDE, and UN Population Division (2022) estimates.
 
-          10,000 BCE - 1799: Historical estimates by HYDE (v3.2).
-          1800-1949: Historical estimates by Gapminder.
-          1950-2021: Population records by the United Nations - Population Division (2022).
-          2022-2100: Projections based on Medium variant by the United Nations - Population Division (2022).
+          - 10,000 BCE - 1799: Historical estimates by HYDE (v3.2). Includes some datapoints Gapminder (Systema Globalis).
```

Comment:
Given that you are using many `-`, maybe it's better to use `+` or `·` or `•` as bullet points. Alternatively, you could use [HTML tags](https://owid.slack.com/archives/C02706JNFC6/p1680185368251889), but I think it's better to avoid that in ETL metadata.